assets/js/base/utils/get-inline-styles.ts

```diff
@@ -0,0 +1,163 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { kebabCase } from 'lodash';
```

Comment:
Probably should avoid the use of lodash here given you've worked on extracting it everywhere?

Reply:
Yeah I was waiting for the other to merge so I can use the new package.